3-platform/shopping-cart/infra/src/main/java/com/_6core/platform/shopping/cart/infra/dto/CreateOrderResponse.java

```diff
@@ -0,0 +1,3 @@
+package com._6core.platform.shopping.cart.infra.dto;
```

Comment:
I dont like name of packages/classes like `dto`. Pls provide more concrete name
```suggestion
package com._6core.platform.shopping.cart.infra.dto;
```